packages/celotool/src/e2e-tests/slashing_tests.ts

```diff
@@ -314,7 +314,7 @@ describe('slashing tests', function(this: any) {
 
       // Penalty is defined to be 5000 cGLD in migrations, locked gold is 10000 cGLD for a validator
       const balance = await lockedGold.getAccountTotalLockedGold(signer)
-      assert.equal(balance.toString(10), '5000000000000000000000')
+      assert.equal(balance.toString(10), '1000000000000000000000')
```

Comment:
Nit: change comment to 9k penalty, and add ~ "so after slashing locked gold is 1000cGld"

Reply:
good catch, thanks!